db/migrate/20171025191957_add_member_to_users.rb

```diff
@@ -5,9 +5,13 @@ def change
       direction.up do
         add_column :users, :member, :boolean, default: false
 
-        User.all.each do |user|
-          user.member = user.is_member?
-          user.save
+        if User.instance_methods.include?(:is_member?)
+
+          User.all.each do |user|
+            user.member = user.is_member?
+            user.save
+          end
+          
         end
       end
 
```

Comment:
As discussed in the team meeting, this one is no longer necessary.

Reply:
I reverted back to prior version of this file.